lib/instrumentation/mysql/mysql.js

```diff
@@ -108,11 +111,11 @@ function wrapGetConnection(shim, connectable) {
   return true
 }
 
-function wrapGetConnectionCallback(shim, cb) {
+exports.wrapGetConnectionCallback = function wrapGetConnectionCallback(shim, cb) {
```

Comment:
Would it make sense to define the module exports together elsewhere in the file, so that these usages wouldn't need the explicit export every time they were used?

Reply:
Sure! I can update to do that instead

Reply:
My thought there was that we could avoid having to refer to each function as `exports.functionName`, but still be able to use the exported functions for testing.